spec/advisory_example.rb

```diff
@@ -154,6 +154,13 @@
         expect(subject).to be_kind_of(Float).or(be_nil)
       end
 
+      case advisory['cvss_v4']
+      when Float
+        context "when a Float" do
+          it { expect(subject).to be_between(0.0, 10.0) }
```

Comment:
Should be `advisory['cvss_v4']` or this logic needs to be moved into it's own `describe` block outside of the `describe "cvss_v3"` block.

Reply:
The previous code also mentioned v2 in this block so I was confused.

Reply:
Made a separate describe block but did include non-v4 code from v3 code.